### PR TITLE
Remove custom imports target

### PR DESCRIPTION
**Pull request by kopiczko**
Towards https://github.com/giantswarm/giantswarm/issues/14992

This is to avoid collisions.

See https://github.com/giantswarm/apiextensions/pull/659

### CODE REVIEW

File: Makefile.custom.mk [hunk 1]
```diff
@@ -113,11 +113,6 @@ generate-fs: $(ESC) config/crd
 	-modtime 0 \
 	config/crd
 
-.PHONY: imports
-imports: $(GOIMPORTS)
```
Comment (tfussell) [inline Makefile.custom.mk:117]: Looks like `goimports` is not being installed now.

Comment (kopiczko) [reply Makefile.custom.mk:117]: Is it OK to assume it should be installed on dev machines?

Comment (kopiczko) [reply Makefile.custom.mk:117]: I see the CI is unhappy.

Comment (kopiczko) [reply Makefile.custom.mk:117]: I renamed the target to `ci-imports` instead.

Comment (kopiczko) [reply Makefile.custom.mk:117]: @tfussell do you think there is a better alternative?

Comment (tfussell) [reply Makefile.custom.mk:117]: This looks fine to me for this repo. But shouldn't `make imports` work in CI as well? Anyways, I'll go ahead and approve.

**Review by tfussell — APPROVED** [review]

**Comment by kopiczko** [comment]
@tfussell do we still need it? I got quite rusty with apiextensions.

**Comment by tfussell** [comment]
> @tfussell do we still need it? I got quite rusty with apiextensions.

We already renamed the target to `local-imports` in a different PR, so I think we can close this.

**Pull request closed without merging**